lib/rouge/guessers/util.rb

```diff
@@ -1,18 +1,30 @@
 module Rouge
   module Guessers
     module Util
+      module SourceNormalizer
+        UTF8_BOM = "\xEF\xBB\xBF"
+        UTF8_BOM_RE = /\A#{UTF8_BOM}/
+
+        # @param [String,nil] source
```

Comment:
This method will no longer accept nor return nil

Reply:
Right! Will fix the doc.